bb/ramfs.go

```diff
@@ -28,10 +28,7 @@ var (
 	// e.g., /lib/modules/4.04, you would do add the root as / and the
 	// starting point for the walk as lib/modules/4.04. That way we only preserve
 	// as much of the path as we need, but we can preserve it all.
-	paths      = map[string][]string{}
-	extraPaths = flag.String("extra", "", `Extra paths to add in the form root:start, e.g. /:etc/hosts.
-The path before the : is used as a starting point for a walk; the path after the : selects what things to put
-into the initramfs starting at /. E.g., /tmp/prototype:/ will install the prototype file system into / of the initramfs`)
+	paths = map[string][]string{}
```

Comment:
It's sufficient to write `map[string][]string` (tbh, I had no idea you could put the `{}` at the end and it threw me off)